src/librssguard/services/standard/standardserviceroot.cpp

```diff
@@ -256,6 +256,23 @@ QList<Message> StandardServiceRoot::obtainNewMessages(Feed* feed,
     mess.m_feedId = feed->customId();
   }
 
+
```

Comment:
So you add datetime feature to "Feeds" table and "Feed" superclass, therefore the data and the feature should (?) be available to all feed types, not just "standard" but also for feeds from other plugins (TT-RSS, Greader, etc.).

But here you actually implement the work logic only into "StandardServiceRoot" which deals only with classic RSS/ATOM feeds.

Did you actually meant to add this feature only for classic RSS/ATOM plugin (like QuiteRSS does since it does not have other plugins/synchronizations) or do you prefer to have this feature unversally for all plugins?